motion.py

```
@@ -400,6 +419,63 @@ def resample_slice_pair(
+    # the input can be either a nifti file or an SITK image
+    if isinstance(ref_img, sitk.Image):
+        ref_img = ref_img
+    elif os.path.isfile(ref_img):
+        ref_img = sitk.ReadImage(ref_img, sitk.sitkFloat32)
+    else:
+        raise ValueError(f'{ref_img} is neither a file nor an SITK image.')
+
+    fixed_upsample = isotropic_upsample_and_pad(ref_img, interpolation)
+
+    if not moving_img.GetDimension()==4:
+        raise ValueError(f"moving_img input must be 4D, got {moving_img.GetDimension()}D instead")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)